src/test/query/notes.integration.ts

```diff
@@ -0,0 +1,74 @@
+import { ApolloServer } from 'apollo-server-express';
+import { getServer } from '../../server';
+import sinon from 'sinon';
+import { ContextManager } from '../../context';
+import { readClient } from '../../database/client';
+import { seedData } from './highlights-fixtures';
+import { noteSeedCommand, GET_NOTES } from './notes-fixtures';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient } from '@aws-sdk/lib-dynamodb';
+import config from '../../config';
+
+describe('Notes on a Highlight', () => {
+  let server: ApolloServer;
+  let contextStub;
+  const userId = 1;
+  const db = readClient();
+  const now = new Date();
+  const testData = seedData(now);
+  const client = new DynamoDBClient({
+    region: config.aws.region,
+    endpoint: config.aws.endpoint,
+  });
+  const dynamodb = DynamoDBDocumentClient.from(client);
+
+  beforeAll(async () => {
+    await Promise.all(
+      Object.keys(testData).map((table) => db(table).truncate())
+    );
+    await Promise.all(
+      Object.entries(testData).map(([table, data]) => db(table).insert(data))
+    );
+    await dynamodb.send(noteSeedCommand(now));
+    contextStub = sinon.stub(ContextManager.prototype, 'userId').value(userId);
+    server = getServer();
+  });
+  afterAll(() => {
+    contextStub.restore();
+    client.destroy();
+  });
+  it('should return a highlight with note when available', async () => {
+    const variables = { itemId: 1 };
+    const res = await server.executeOperation({
+      query: GET_NOTES,
+      variables,
+    });
+
+    const expectedHighlightWithNote = {
+      id: '1',
+      note: {
+        _createdAt: Math.round(now.getTime() / 1000),
```

Comment:
seen this a couple of times now, let's make it a utility function the next time we see it.

Reply:
why it already doesn't exist... idk